check_patroni/cli.py

```diff
@@ -369,9 +377,10 @@ def cluster_has_replica(
     critical: str,
     sync_warning: str,
     sync_critical: str,
+    sync_type: Literal["any", "sync", "quorum"],
```

Comment:
I'd also suggest to define a `SyncType = Literal["any", "sync", "quorum"]` variable once somewhere (e.g. in the `types` module) and reuse it here and everywhere else to avoid repeating the definition.